<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>674</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>674</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>JACK Control</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/audio_out.svg</normaloff>:/images/audio_out.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <widget class="QWidget" name="tabConfiguration">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/audio-card.svg</normaloff>:/images/audio-card.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Configuration</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="JackPresetsWidget" name="widget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabConnections">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/emblem-generic.svg</normaloff>:/images/emblem-generic.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Connections</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="ConnectionsWidget" name="connectionsWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabMessages">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/gtk-info.svg</normaloff>:/images/gtk-info.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Messages</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTextEdit" name="messagesTextEdit">
          <property name="font">
           <font>
            <family>Ubuntu Mono</family>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabStatus">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/gnome-nettool.svg</normaloff>:/images/gnome-nettool.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Status</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="StatusWidget" name="statusWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBarTransport">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea|Qt::LeftToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTransportRewind"/>
   <addaction name="actionTransportBackward"/>
   <addaction name="actionTransportStart"/>
   <addaction name="actionTransportStop"/>
   <addaction name="actionTransportForward"/>
  </widget>
  <widget class="QToolBar" name="toolBarJackServer">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStopJackServer"/>
   <addaction name="actionStartJackServer"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionTransportStart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/play1.png</normaloff>:/images/play1.png</iconset>
   </property>
   <property name="text">
    <string>Transport Start</string>
   </property>
   <property name="toolTip">
    <string>Transport Start</string>
   </property>
   <property name="statusTip">
    <string>Transport Start</string>
   </property>
   <property name="whatsThis">
    <string>Transport Start</string>
   </property>
  </action>
  <action name="actionTransportStop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/pause1.png</normaloff>:/images/pause1.png</iconset>
   </property>
   <property name="text">
    <string>Transport Stop</string>
   </property>
   <property name="toolTip">
    <string>Transport Stop</string>
   </property>
   <property name="statusTip">
    <string>Transport Stop</string>
   </property>
   <property name="whatsThis">
    <string>Transport Stop</string>
   </property>
  </action>
  <action name="actionTransportBackward">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/backward1.png</normaloff>:/images/backward1.png</iconset>
   </property>
   <property name="text">
    <string>Transport Backward</string>
   </property>
   <property name="toolTip">
    <string>Transport Backward</string>
   </property>
   <property name="statusTip">
    <string>Transport Backward</string>
   </property>
   <property name="whatsThis">
    <string>Transport Backward</string>
   </property>
  </action>
  <action name="actionTransportForward">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/forward1.png</normaloff>:/images/forward1.png</iconset>
   </property>
   <property name="text">
    <string>Transport Forward</string>
   </property>
   <property name="toolTip">
    <string>Transport Forward</string>
   </property>
   <property name="statusTip">
    <string>Transport Forward</string>
   </property>
   <property name="whatsThis">
    <string>Transport Forward</string>
   </property>
  </action>
  <action name="actionTransportRewind">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/rewind1.png</normaloff>:/images/rewind1.png</iconset>
   </property>
   <property name="text">
    <string>Transport Rewind</string>
   </property>
   <property name="toolTip">
    <string>Transport Rewind</string>
   </property>
   <property name="statusTip">
    <string>Transport Rewind</string>
   </property>
   <property name="whatsThis">
    <string>Transport Rewind</string>
   </property>
  </action>
  <action name="actionStartJackServer">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/start1.png</normaloff>:/images/start1.png</iconset>
   </property>
   <property name="text">
    <string>Start JACK Server</string>
   </property>
   <property name="toolTip">
    <string>Start JACK Server</string>
   </property>
   <property name="statusTip">
    <string>Start JACK Server</string>
   </property>
   <property name="whatsThis">
    <string>Start JACK Server</string>
   </property>
  </action>
  <action name="actionStopJackServer">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/stop1.png</normaloff>:/images/stop1.png</iconset>
   </property>
   <property name="text">
    <string>Stop JACK Server</string>
   </property>
   <property name="toolTip">
    <string>Stop JACK Server</string>
   </property>
   <property name="statusTip">
    <string>Stop JACK Server</string>
   </property>
   <property name="whatsThis">
    <string>Stop JACK Server</string>
   </property>
  </action>
  <action name="actionJackSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/preferences-desktop.svg</normaloff>:/images/preferences-desktop.svg</iconset>
   </property>
   <property name="text">
    <string>JACK Settings</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StatusWidget</class>
   <extends>QWidget</extends>
   <header>statuswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConnectionsWidget</class>
   <extends>QWidget</extends>
   <header>connectionswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JackPresetsWidget</class>
   <extends>QWidget</extends>
   <header>jackpresetswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
